components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/utils/LoggerUtils.java

```diff
@@ -55,6 +60,10 @@ public class LoggerUtils {
     private static final String CORRELATION_ID_MDC = "Correlation-ID";
     private static final String FLOW_ID_MDC = "Flow-ID";
     private static final String CLIENT_COMPONENT = "clientComponent";
+    private static final Set<String> LOGGABLE_PARAMS = Collections.unmodifiableSet(
+            new HashSet<>(Arrays.asList("grant_type", "redirect_uri", "client_id", "clientId", "username", "scope",
+                    "audience", "response_type", "state", "nonce", "user", "redirectUrl", "requestedScopes",
+                    "redirectUri", "authenticatedUser", "appState")));
```

Comment:
Can you please review these entries that will be whitelisted? @pulasthi7 @hasinthaindrajee @mefarazath @senthalan

Reply:
won't the authenticatedUser have sensitive user details like claims? I think it is better to omit this.

Reply:
can we add user id as well ?